container-images/tcib/base/os/tempest/run_tempest.sh

```diff
@@ -170,6 +170,27 @@ if [[ ! -z ${TEMPESTCONF_REMOVE} ]]; then
     done <<< "$TEMPESTCONF_REMOVE"
 fi
 
+if [[ ! -z ${TEMPEST_WHITEBOX_NEUTRON_IMAGE_URL} ]]; then
```

Comment:
Let's put it into a separate function called for example `generate_extra_tempest_configuration`